packages/frontend/package.json

```diff
@@ -30,17 +30,27 @@
     "@reduxjs/toolkit": "^2.2.1",
```

Comment:
**issue (blocking):** Please, review this file and ensure that you installed the right packages and in their appropriate sections, for example, `commitlint` is installed in the root `package.json` and doesn't belong here. Also, we have the dependencies and devDependencies sections. Dependencies are for packages that are used in development and production while devDependencies are packages used only during development and I believe the leaflet packages are required for proper functioning of the app in production. Therefore, kindly review and ensure you installed packages correctly and in their appropriate sections. Thank you

Reply:
are you saying that the leaflet stuff needs to be in both the Dependencies and devDependencies or just Dependencies?

Reply:
The leaflet stuff needs to be in only the dependencies. Also, we are using a monorepo (lerna) and so dependencies shared by packages (frontend and backend) like `commitlint` should only be in the root package.json and nowhere else. Please refer to older version of the app to get a clearer picture of this. Thank you